### PR TITLE
remove direct dependency on github.com/pkg/errors

### PR DESCRIPTION
**Pull request by AkihiroSuda**
Used https://github.com/AkihiroSuda/go-wrap-to-percent-w for converting `errors.Wrap` to `%w`

### CODE REVIEW

File: pkg/cidata/template.go [hunk 1]
```diff
@@ -3,15 +3,15 @@ package cidata
 import (
 	"bytes"
 	"embed"
-	_ "embed"
```
Comment (AkihiroSuda) [inline pkg/cidata/template.go:6]: (This fix is unrelated to pkg/errors)

**Review by jandubois — APPROVED** [review]
LGTM
